app/src/main/java/com/concordium/wallet/uicore/dialog/AuthenticationDialogFragment.kt

```diff
@@ -76,6 +78,15 @@ class AuthenticationDialogFragment : DialogFragment(), TextView.OnEditorActionLi
             callback?.onCancelled()
             dismiss()
         }
+        binding.includeDialogAuthenticationContent.passwordEdittext.requestFocus()
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        return super.onCreateDialog(savedInstanceState).apply {
+            // The correct input mode needs to be set
+            // for the keyboard to be shown automatically.
+            window?.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE)
```

Comment:
I don't know how to verify that this actually works as I don't have a phone without biometrics available for testing.